sos/plugins/openstack_manila.py

```diff
@@ -51,12 +84,8 @@ def apply_regex_sub(self, regexp, subst):
         )
 
     def postproc(self):
-        protect_keys = [
-            "nova_admin_password",  "rabbit_password",  "qpid_password",
-            "password", "netapp_nas_password", "cinder_admin_password",
-            "neutron_admin_password", "service_instance_password",
-            "transport_url"
-        ]
+        protect_keys = [".*password.*", "transport_url",
+                        "hdfs_ssh_pw", "maprfs_ssh_pw"]
```

Comment:
On the surface this looks good to me, but as a sanity check measure can you provide an example of this properly scrubbing keys here before we merge?

Reply:
Yep, here it is: http://paste.openstack.org/show/763919/